### DISCUSSION TITLE
React Native Camera but With React Hooks - Jagad Yudha Awali

### DISCUSSION THREAD

**Discussion by giscus** [jagadyudha/jagad.dev#28] [Announcements]
# React Native Camera but With React Hooks - Jagad Yudha Awali

Control the camera and interacts with the device's native operating system and hardware using RNCamera

https://jagad.xyz/posts/react-native-camera-but-with-react-hooks

**Comment by jagadyudha** [comment]
Please leave comments and feedback on this post 😀
